mp/src/game/shared/momentum/run/mom_replay_factory.cpp

```diff
@@ -70,6 +71,28 @@ CMomReplayBase* CMomReplayFactory::LoadReplayFile(const char* pFileName, bool bF
         return nullptr;
     }
 
+    return LoadReplayFromBuffer(reader, bFullLoad);
+}
+
+CMomReplayBase* CMomReplayFactory::LoadReplayFromBuffer(CUtlBuffer& reader, bool bFullLoad)
```

Comment:
Can the buffer be passed as a const reference?

Reply:
not without changing CMomReplayV1, at least